### PR TITLE
Added Golang example

### PR DESCRIPTION
**Pull request by WestleyK**
Not very practical, but it shows what can be done with a container.

### CODE REVIEW

File: lang/golang/README.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+
+
+## Setup:
+
+First, make the working directory:
+
+```
+$ mkdir golang-container
+$ cd goland-container/
```
Comment (jchavezlavalle) [inline lang/golang/README.md]: Here it should be `$ cd golang-container/`

File: lang/golang/golang.def [hunk 2]
```diff
@@ -0,0 +1,11 @@
+Bootstrap: docker
+From: golang:1.11.1
+
+%help
+usage: ./golang.sif [go option] <YOUR_GO_SOURCE_CODE>
+
+  examples:
+  $ ./golang.sif build hello-world.go
```
Comment (jchavezlavalle) [inline lang/golang/golang.def:8]: fix indentation here

File: lang/golang/hello-world.go [hunk 3]
```diff
@@ -0,0 +1,9 @@
+package main
+
+import "fmt"
+
+func main() {
+    fmt.Print("hello world!\n")
+
```
Comment (jchavezlavalle) [inline lang/golang/hello-world.go]: avoid blank line 7 inside the `func main()`

**Review by jchavezlavalle — APPROVED** [review]
lgtm :+1:

File: lang/golang/README.md [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+```
+
+The `golang.sif` is basically the `go` command, running in the container.
+
+<br>
+
+Adding `golang.sif` to `$PATH`:
+
+```
+$ export PATH=$PATH:/$HOME/golang-container/
```
Comment (jmstover) [inline lang/golang/README.md]: Change `/$HOME` -> `$HOME`

File: lang/golang/README.md [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+
+Now you should have `golang.sif` command.
+
+<br>
+
+## Building the container from a definition file:
+
+You will need root access when building from a definition file.
+
+Here's the [definition file](https://raw.githubusercontent.com/sylabs/examples/master/lang/golang/golang.def), you can modify this as you need.
```
Comment (jmstover) [inline lang/golang/README.md]:
```
Here's the definition file, **which can be modified as needed**.
```

**Review by jmstover — CHANGES_REQUESTED** [review]

**Review by jmstover — APPROVED** [review]

**Pull request merged**